ush/forecast_predet.sh

```diff
@@ -332,27 +332,27 @@ FV3_predet(){
     if [[ "${TYPE}" == "nh" ]]; then  # monotonic and non-hydrostatic
       hord_mt=${hord_mt_nh_mono:-"10"}
       hord_xx=${hord_xx_nh_mono:-"10"}
-      hord_dp=-${hord_xx_nh_nonmono:-"-10"}
+      hord_dp=${hord_xx_nh_mono:-"10"}
     else  # monotonic and hydrostatic
       hord_mt=${hord_mt_hydro_mono:-"10"}
       hord_xx=${hord_xx_hydro_mono:-"10"}
-      hord_dp=-${hord_xx_nh_nonmono:-"-10"}
+      hord_dp=${hord_xx_hydro_mono:-"10"}
+      kord_tm=${kord_tm_hydro_mono:-"-12"}
+      kord_mt=${kord_mt_hydro_mono:-"12"}
+      kord_wz=${kord_wz_hydro_mono:-"12"}
+      kord_tr=${kord_tr_hydro_mono:-"12"}
     fi
   else  # non-monotonic options
     d_con=${d_con_nonmono:-"1."}
     do_vort_damp=".true."
     if [[ "${TYPE}" == "nh" ]]; then  # non-monotonic and non-hydrostatic
       hord_mt=${hord_mt_nh_nonmono:-"5"}
       hord_xx=${hord_xx_nh_nonmono:-"5"}
-      hord_dp=${hord_xx_hydro_mono:-"-5"}
+      hord_dp=${hord_xx_nh_nonmono:-"-5"}
```

Comment:
it should be changed to hord_dp=${hord_dp_nh_nonmono:-"-5"}.
Then add "export hord_dp_nh_nonmono=-5" in parm/config/gfs/config.fcst